php.ini-production

```diff
@@ -1374,16 +1416,14 @@ session.gc_probability = 1
 ; the gc will run on any give request. Increasing this value to 1000 will give you
 ; a 0.1% chance the gc will run on any give request. For high volume production servers,
 ; this is a more efficient approach.
-; Default Value: 100
-; Development Value: 1000
-; Production Value: 1000
 ; http://php.net/session.gc-divisor
```

Comment:
What's the reason for deleting these lines? You can just update their values instead.